### PR TITLE
Update IMPORT_EXISTING_DOCUMENTS.md

### PR DESCRIPTION
**Pull request by dipbhi**
- Importing data with COLLECTION_GROUP_QUERY needs a clear example. It's very confusing without one.
- "The import script uses several values from your installation of the extension" gives an impression that the script will figure it out from the installed extension. The word "requires" may be better suited.

### CODE REVIEW

File: firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ You may pause and resume the import script from the last batch at any point.
...
   - If you run the import script multiple times over the same collection.
 
-- You cannot use wildcard notation in the collection path (i.e. `/collection/{document}/sub_collection}`). Instead, you can use a [collectionGroup](https://firebase.google.com/docs/firestore/query-data/queries#collection-group-query) query. To use a `collectionGroup` query, provide the collection name value as `${COLLECTION_PATH}`, and set `${COLLECTION_GROUP_QUERY}` to `true`.
+- You cannot use wildcard notation in the collection path (i.e. `/collection/{document}/sub_collection}`). Instead, you can use a [collectionGroup](https://firebase.google.com/docs/firestore/query-data/queries#collection-group-query) query. To use a `collectionGroup` query, provide the collection name value as `${COLLECTION_PATH}`, and set `${COLLECTION_GROUP_QUERY}` to `true`. For example, if you are trying to import `/collection/{document}/sub_collection}`, value for the `${COLLECTION_PATH}` should be provided as `sub_collection`. Keep in mind that if you have another sub collection with the same name (e.g. `/collection2/{document}/sub_collection}`, that will be imported too.
```
Comment (cabljac) [inline firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md:20]:
```suggestion
- You cannot use wildcard notation in the collection path (i.e. `/collection/{document}/sub_collection}`). Instead, you can use a [collectionGroup](https://firebase.google.com/docs/firestore/query-data/queries#collection-group-query) query. To use a `collectionGroup` query, provide the collection name value as `${COLLECTION_PATH}`, and set `${COLLECTION_GROUP_QUERY}` to `true`. For example, if you are trying to import `/collection/{document}/sub_collection`, the value for the `${COLLECTION_PATH}` should be provided as `sub_collection`. Keep in mind that if you have another sub collection with the same name (e.g. `/collection2/{document}/sub_collection`, that will be imported too.
```

**Comment by cabljac** [comment]
Hi there! Thanks, this looks good to me, added one small comment.

**Comment by dipbhi** [comment]
Hi @cabljac,
Just checking if there is any concern stopping you from approving this merge request.

**Comment by cabljac** [comment]
Hey, my only concern is that it will conflict/be overwritten by the changes in 
https://github.com/firebase/extensions/pull/1464/files

**Comment by dipbhi** [comment]
Please let me know once your changes are merged. I will apply my updates again.

**Review by dackers86 — APPROVED** [review]

**Pull request merged**